core/store/migrate/migrations/0235_add_workflow_models.sql

```diff
@@ -0,0 +1,30 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE workflow_executions (
+	id varchar(64) PRIMARY KEY,
+	workflow_id varchar(64) NOT NULL,
+	status text NOT NULL
+);
+
+CREATE TABLE workflow_steps (
+	id SERIAL PRIMARY KEY,
```

Comment:
@HenryNguyen5 Wdym by this?

Reply:
> @HenryNguyen5 Wdym by this?

Should we be storing timestamps of when a step started and ended? I'd imagine this will become important when working with workflow rehydration, timeouts, and displaying historical workflow executions .

Reply:
Oh yep -- I've added some